stable/node-local-dns/values.yaml

```diff
@@ -19,6 +19,18 @@ config:
   # Port used for the health endpoint
   healthPort: 8080
 
+  setupInterface: {}
+
+  setupIptables: {}
+
+  skipTeardown: {}
```

Comment:
I think these are not maps, they are simply booleans, no? e.g. `-setupiptables=false`

Reply:
Yes, they are boolean. Changed. Can you check please?

Reply:
Hi @max-rocket-internet. Any news?

Reply:
The README hasn't been updated with the new boolean values